projects/plugins/super-cache/wp-cache.php

```diff
@@ -3294,7 +3323,8 @@ function wp_cron_preload_cache() {
 			$page_on_front = $page_for_posts = 0;
 		}
 		if ( $wp_cache_preload_email_me && $wp_cache_preload_email_volume == 'many' )
-			wp_mail( get_option( 'admin_email' ), sprintf( __( '[%1$s] Refreshing posts from %2$d to %3$d', 'wp-super-cache' ), home_url(), $c, ($c+100) ), ' ' );
+		/* translators: 1: home url, 2: start post id, 3: end post id */
+			wp_mail( get_option( 'admin_email' ), sprintf( __( '[%1$s] Refreshing posts from %2$d to %3$d', 'wp-super-cache' ), home_url(), $c, ( $c + 10 ) ), ' ' );
```

Comment:
Might be worth making a constant / define / variable which sets the size of preloads, so that:
a) Each instance of `10` or `100` in code is easier to understand because it will be a label like `PRELOAD_POST_COUNT`, and 
b) Any changes in future to preload size become easier.

Reply:
Good idea. Just added that!